app/templates/login.hbs

```diff
@@ -124,37 +126,41 @@
     onClose=closeModal
     translucentOverlay=true
   }}
-    <h2>Forgot password</h2>
+    <h2 class="modal-header">Forgot password</h2>
     {{#let completeForgotPassword as |task|}}
       <form onsubmit={{prevent-default (perform task)}}>
         <legend>
           You should receive an email with a confirmation code. Enter it here and
           choose a new password
         </legend>
         <div>
-          <label>code:</label>
+          {{!-- <label>code:</label> --}}
```

Comment:
Is there a reason you're leaving this comment here?

Reply:
As above:)